content/applications/inventory_and_mrp/quality/quality_reporting/quality_alerts_report.rst

```diff
@@ -0,0 +1,210 @@
+=====================
+Quality alerts report
+=====================
+
+In Odoo's *Quality* app, the quality alerts report displays key statistics about existing alerts.
+This includes the number of quality alerts in each stage, the number of alerts assigned to each
+quality team, the root causes of the issues that led to alert creation, and more.
+
+The report offers a number of options to help visualize the data, including customizable views and
+three different graph types. This allows users to extract deeper insights from their quality alert
+processes.
+
+Report dashboard
+================
+
+To access the quality alerts report dashboard, navigate to :menuselection:`Quality app --> Reporting
+--> Quality Alerts`.
+
+There are five main views that can be used to display quality alert data:
+
+- :ref:`Graph view <quality_alerts_report/graph-view>`
+- :ref:`Pivot view <quality_alerts_report/pivot-view>`
+- :ref:`Kanban view <quality_alerts_report/kanban-view>`
+- :ref:`List view <quality_alerts_report/list-view>`
+- :ref:`Calendar view <quality_alerts_report/calendar-view>`
```

Comment:
You may want to consider adding in the appropriate icons beside each of these, so the reader can know what the view icons represent. just a thought.